Makefile

```diff
@@ -16,7 +16,7 @@ gobuild: go-check
 CATALOG_REGISTRY_ORGANIZATION?=app-sre
 
 .PHONY: skopeo-push
-skopeo-push:
+skopeo-push: docker-build
```

Comment:
In [app_sre_build_deploy.sh](https://github.com/openshift/custom-domains-operator/blob/21b782149ddb46983a7faae62e2f9858a3406e77/hack/app_sre_build_deploy.sh#L12), you should either
- remove `build`; or
- `s/build/docker-build/` and then you don't need this delta.

Reply:
https://github.com/openshift/custom-domains-operator/pull/18 does the latter.